www/source/blog/2017-06-14-Habitat-Issue-Tracker.html.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Habitat.sh Governance and Issue Tracking
+date: 2017-06-14
+author: Ian Henry
+tags: Governance,Community
+category: Community
+classes: body-article
+---
+Hopefully our previous [blog](https://www.habitat.sh/blog/2017/06/Triage-and-Contributions/) and [discourse]https://forums.habitat.sh/t/habitat-community-triage/287) posts have given you some general understanding around the way we govern the Habitat project. But, as a group of people hoping to share what we're building with a larger audience we know we can always be doing more to "pull back the curtain". We can be doing more to [expose our work](https://ext.prodpad.com/ext/roadmap/d2938aed0d0ad1dd62669583e108357efd53b3a6) and keep our community informed and educated on the ways we've chosen to operate.
+
+Thus, I'm here to explain as much as I can about the way we currently govern the Habitat Project and how the Issue Tracker fits in. Generally this blog post is going to build on top of the Triage and Contribution blog post linked
+
+## Core Team
+Habitat is governed by a core team, which is ultimately responsible for all decision-making in the project. Specifically, the core team alongside a Chef Inc. Product Owner:
+* Sets the overall direction and vision for the project
```

Comment:
This bullet section needs to be offset by empty lines in order to get the formatting to work. (Right now it gets mashed into a single unformatted line.)